Automattic/jetpack
# Get Site Endpoint: remove unused difm references

Fixes #

## Proposed changes:

* This PR removes the `difm_lite_site_options` key from the site options array of the `https://public-api.wordpress.com/rest/v1/sites/<site id>` endpoint.
* This key was relevant to WPCOM only and is not used in Calypso anymore. It was removed via https://github.com/Automattic/wp-calypso/pull/84585 

### Other information:


## Jetpack product discussion


NA

## Does this pull request change what data or activity we track or use?


NA

## Testing instructions:






* Test on a simple site by following the command in the first comment below.
* Sandbox public-api.wordpress.com
* Go to `https://wordpress.com/home/<site slug>`.
* In the browser Network tab, confirm that the request to `https://public-api.wordpress.com/rest/v1.1/sites/<blog id>?http_envelope=1&force=wpcom` is successful.
* Confirm that there are no PHP errors on the sandbox.
